docs/dashboard_config.md

```diff
@@ -75,13 +81,13 @@ The `notebookController` field controls the Notebook Controller options such as
 
 ```yaml
 notebookController:
-    enabled: true
-    envVarConfig:
-        enabled: true
-    gpuConfig:
-        enabled: true
-    pvcSize: 20Gi
-    notebookNamespace: odh-notebooks
+  enabled: true
+  envVarConfig:
+      enabled: true
+  gpuConfig:
+      enabled: true
```

Comment:
Looks like some of this is not around anymore.
https://github.com/opendatahub-io/odh-dashboard/blob/main/frontend/src/types.ts#L44-L59

Can we clean up the docs for accuracy with this PR?